src/content/releases/_index.en.md

```diff
@@ -1,10 +1,26 @@
 +++
-date = 2020-05-04T16:50:16+02:00
+date = 2020-08-24T11:35:16+02:00
 title = "Releases"
 pre = "<b>5. </b>"
 weight = 25
 +++
 
+## v0.6.0 Improved Submariner High Availability and various Lighthouse enhancements
+
+> This release mainly focused on support for headless Services in Lighthouse,
+> as well as improving Submariner's High Availability (HA).
+
+* **Lighthouse** has been enhanced to:
```

Comment:
@mangelajo Are you planning to include that "Service Discovery" will be enabled by default while joining clusters?

Reply:
L30 :-) may be I shall move it to this section instead?

Reply:
Done.